recipes/freetype/build.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+./configure --prefix=$PREFIX \
+            --with-zlib=$PREFIX \
+            --with-png=yes \
+            --with-bzip2=no \
+            --without-harfbuzz
```

Comment:
Also, there is a more recent [one]( https://github.com/ContinuumIO/anaconda-recipes/tree/master/harfbuzz ) in anaconda-recipes.